multi-node/aws/pkg/config/config.go

```diff
@@ -74,9 +74,27 @@ func ClusterFromBytes(data []byte) (*Cluster, error) {
 	// as it will with RecordSets
 	c.HostedZone = WithTrailingDot(c.HostedZone)
 
+	// If the user specified no subnets, we assume that a single AZ configuration with the default instanceCIDR is demanded
+	if len(c.Subnets) == 0 && c.InstanceCIDR == "" {
+		c.InstanceCIDR = "10.0.0.0/24"
+	}
+
 	if err := c.valid(); err != nil {
 		return nil, fmt.Errorf("invalid cluster: %v", err)
 	}
+
+	// For backward-compatibility
+	if len(c.Subnets) == 0 {
+		if c.InstanceCIDR == "" {
+			c.InstanceCIDR = "10.0.0.0/24"
```

Comment:
I think this `if c.InstanceCIDR` block is unnecessary since it will have been done before the call to valid(), and valid shouldn't change the value of `c.InstanceCider`.